lib/api/src/grpc/proto/points.proto

```diff
@@ -229,6 +229,21 @@ message SearchBatchPoints {
   optional ReadConsistency read_consistency = 3; // Options for specifying read consistency guarantees
 }
 
+message WithLookup {
+  string collection = 1; // Name of the collection to use for points lookup
+  WithPayloadSelector with_payload = 2; // Options for specifying which payload to include (or not)
+  WithVectorsSelector with_vectors = 3; // Options for specifying which vectors to include (or not)
+}
+
+// Interface for specifying lookup collection, intended for groups API only.
+// This allows specifying the collection name only, or the collection name and payload/vectors options.
+message WithLookupInterface {
```

Comment:
are you sure we want this type of sugar for gRPC? `oneof` structures in generated code are kinda ugly, maybe we can keep the plain `WithLookup` with optional with_payload and with_vectors?

Reply:
I did it to keep the same structure as in the openapi spec, but it's true it might be too much for grpc

Reply:
Removed